### PR TITLE
Feature/orange 885 va formulary

### PR DESCRIPTION
**Pull request by orndorffgrant**
# What does this do?
Two things
1. Introduces a script for importing VA Formulary data from a spreadsheet and inserting it into mongodb. The excel sheet located here (https://www.pbm.va.gov/PBM/nationalformulary/VANF-October_2018.xls) was used for developing the script.
    * The script was developed with the assumption that the general format of the excel sheet won't change in specific ways.
2. Adds an endpoint for searching through this new collection of the va formulary entries

# How do I test it?
* run the script with the file linked to above as the only argument. Check mongo to make sure the entries got added.
* using postman, GET /formulary/search?genericName=SEARCH_TERM_HERE
  * try searching for different entries that you see in the spreadsheet

### CODE REVIEW

**Comment by orndorffgrant** [comment]
Failing tests appear to be unrelated to this PR. In fact, they look related to Daylight Savings Time, which would be weird...

File: import_va_formulary.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+const mongoose = require('mongoose');
+const config = require("./config.js");
+
+const keypress = async () => {
+  console.log('\tPress any key to continue or Control-C to cancel.\n');
+  process.stdin.setRawMode(true);
+  return new Promise(resolve => process.stdin.once('data', data => {
+    const byteArray = [...data];
+    if (byteArray.length > 0 && byteArray[0] === 3) {
+      console.log('^C');
```
[the remark this answers is not in the document]
Comment (orndorffgrant) [reply import_va_formulary.js]: Disclaimer: I got this snippet off Stack Overflow.

Just for some visual feedback to the user. I think the normal way control-c is handled is by the shell catching it, printing "^C", and sending a sigint to the running program. But I guess when we're in raw input mode on stdin, we catch the control-c and the shell doesn't see it or print "^C", so we print it so the user knows their control-c worked.

**Comment by orndorffgrant** [comment]
Good ideas! I'll add the package.json script and some docs.

File: import_va_formulary.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (dbrezack) [reply import_va_formulary.js]: Fair enough 👍

**Pull request merged**